src/list.scss

```diff
@@ -11,6 +11,53 @@
     .fd-list+(__item, __label, _input,  __title, __message, __footer)+(--no-wrap)
 */
 
+$block: #{$fd-namespace}-list;
+$form-item: #{$fd-namespace}-form-item;
+
+$fd-list-item-padding-x: 1rem;
+$fd-dropdown-vertical-padding: 0.75rem;
+$fd-dropdown-vertical-compact-padding: 0.5rem;
```

Comment:
no `!default` ?

Reply:
I just moved the existing variables outside of the block, didn't notice they were missing the `!default`. I addded it to all vars that were missing it. Thanks for noticing it.